README.md

```diff
@@ -16,37 +19,44 @@ master    | [![CircleCI](https://circleci.com/gh/tendermint/tendermint/tree/mast
 _NOTE: This is yet pre-alpha non-production-quality software._
 
 Tendermint Core is Byzantine Fault Tolerant (BFT) middleware that takes a state transition machine, written in any programming language,
-and replicates it on many machines.
-See the [application developers guide](https://github.com/tendermint/tendermint/wiki/Application-Developers) to get started.
+and securely replicates it on many machines.
+
+For more background, see the [introduction](https://tendermint.com/intro).
+
+To get started developing applications, see the [application developers guide](https://tendermint.com/docs/guides/app-development).
+
+## Install
+
+`go get -u github.com/tendermint/tendermint/cmd/tendermint`
+
+For more details (or if it fails), see the [install guide](https://tendermint.com/intro/getting-started/install).
 
 ## Contributing
 
-Yay open source! Please see our [contributing guidelines](https://github.com/tendermint/tendermint/wiki/Contributing).
+Yay open source! Please see our [contributing guidelines](https://tendermint.com/guides/contributing).
 
 ## Resources
 
 ### Tendermint Core
 
-- [Introduction](https://github.com/tendermint/tendermint/wiki/Introduction)
-- [Validators](https://github.com/tendermint/tendermint/wiki/Validators)
-- [Byzantine Consensus Algorithm](https://github.com/tendermint/tendermint/wiki/Byzantine-Consensus-Algorithm)
-- [Block Structure](https://github.com/tendermint/tendermint/wiki/Block-Structure)
-- [RPC](https://github.com/tendermint/tendermint/wiki/RPC)
-- [Genesis](https://github.com/tendermint/tendermint/wiki/Genesis)
-- [Configuration](https://github.com/tendermint/tendermint/wiki/Configuration)
-- [Light Client Protocol](https://github.com/tendermint/tendermint/wiki/Light-Client-Protocol)
-- [Roadmap for V2](https://github.com/tendermint/tendermint/wiki/Roadmap-for-V2)
+- [Introduction](https://tendermint.com/intro)
+- [Docs](https://tendermint.com/docs)
+- [Software using Tendermint](https://tendermint.com/ecosystem)
 
 ### Sub-projects
 
 * [TMSP](http://github.com/tendermint/tmsp)
+* [Ethermint](http://github.com/tendermint/ethermint)
```

Comment:
I think these 2 should go under More:  they're not "sub-projects"/dependencies of Tendermint.  Could be confusing, e.g. people thinking Tendermint depends on the EVM